sdv/multi_table/base.py

```diff
@@ -359,3 +360,48 @@ def get_learned_distributions(self, table_name):
         """
         synthesizer = self._table_synthesizers[table_name]
         return synthesizer.get_learned_distributions()
+
+    def add_constraints(self, constraints):
+        """Add constraints to the synthesizer.
+
+        Args:
+            constraints (list):
+                List of constraints described as dictionaries in the following format:
+                    * ``constraint_class``: Name of the constraint to apply.
+                    * ``table_name``: Name of the table where to apply the constraint.
+                    * ``constraint_parameters``: A dictionary with the constraint parameters.
+
+        Raises:
+            SynthesizerInputError:
+                Raises when the ``Unique`` constraint is passed.
+        """
+        if self._fitted:
+            warnings.warn(
+                "For these constraints to take effect, please refit the synthesizer using 'fit'."
+            )
+
+        for constraint in constraints:
+            if constraint['constraint_class'] == 'Unique':
+                raise SynthesizerInputError(
+                    "The constraint class 'Unique' is not currently supported."
+                    'Please remove the constraint for this synthesizer.'
+                )
```

Comment:
Could we pull this in a function before this loop, to ensure that no other constraint was added to the data processor.  Currently we have no way to remove or reset the constraints, @amontanez24 maybe we can add something for it ?

Reply:
That may be something we add later

Reply:
I agree though that if we can't add this constraint, we should either add all of the other ones, or none of the other ones. We don't want to crash half way through